source/config/configSpec.py

```diff
@@ -169,6 +169,7 @@
 	reportBorderStyle = boolean(default=False)
 	reportBorderColor = boolean(default=False)
 	reportLinks = boolean(default=true)
+	reportGraphics = boolean(default=true)

```

Comment:
This should have a capital T I think.

Reply:
With regard to capitals in booleans, the config spec is a real mess. I decided to try changing false to False and true to True in the whole spec, but that resulted into 75 changed lines.

Sounds like something that could be done as part of #9589.

Reply:
We have had this discussion previously in #6090. In short, I would prefer we only make these changes while the feature is changed so that it gets focused testing. That said changing lowercase to uppercase is probably quite safe, someone would have to confirm it though. If it could be confirmed that there was no change in behaviour in terms of the our config package, then I think we would accept that change.